### PR TITLE
feat: Roadmap

### PR DESCRIPTION
**Pull request by dherault**
## Summary

This PR adds a roadmap screen to the application.

You can check it by clicking the roadmap button on the sidebar.

Also: Updates honorable to v1. No breaking change for Plural.

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by dherault** [comment]
I added the roadmap label to test my code using this PR.

**Comment by dherault** [comment]
Required before tests can pass: https://github.com/pluralsh/design-system/pull/381

File: www/src/components/roadmap/RoadmapIssue.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          Requested by: {issue.author}
+        </P>
+      )}
+    </Div>
+  )
+}
+
+const toReplaces = [
+  /<img .*>/g,
+  /^<!--.*-->/g,
```
[the remark this answers is not in the document]
Comment (dherault) [reply www/src/components/roadmap/RoadmapIssue.tsx:115]: There is no newline to match before I use it on single lines.

File: www/package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "@nivo/core": "0.80.0",
     "@nivo/geo": "0.80.0",
     "@nivo/line": "0.80.0",
-    "@octokit/core": "4.1.0",
+    "@octokit/core": "^4.1.0",
```
Comment (maciaszczykm) [inline www/package.json:42]: This should be reverted.

File: www/vite.config.ts [hunk 3]
```diff
@@ -28,21 +29,25 @@ export default defineConfig({
...
-        // @ts-expect-error
-        GlobalPolyFill.default({
-          process: true,
-          buffer: true,
-        }),
+        nodePolyfills,
       ],
     },
   },
+  // optimizeDeps: {
```
Comment (floreks) [inline www/vite.config.ts:38]: The build seems to pass. Can we remove this?

File: www/src/components/roadmap/RoadmapSearchBox.tsx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  issues: IssueType[]
+  displayAuthor?: boolean
+  displayVotes?: boolean
+  displayProgress?: boolean
+  removeVoteSorting?: boolean
+  removeStateSorting?: boolean
+}
+
+const fuseOptions = {
+  // isCaseSensitive: false,
```
Comment (floreks) [inline www/src/components/roadmap/RoadmapSearchBox.tsx:31]: Cleanup unused options.

**Comment by floreks** [comment]
Codewise LGTM. Some leftover code can be removed. If there are no more design comments from the @pluralsh/design team, we should be good to merge.

File: www/src/components/account/AccountAttributes.tsx [hunk 5]
```diff
@@ -81,6 +81,7 @@ function DomainMapping({
               onClick={() => {
                 setConfirm(true)
               }}
+              // @ts-expect-error
```
Comment (DavidSpek) [inline www/src/components/account/AccountAttributes.tsx:84]: Why is this comment needed? A similar comment recently caused the frontend release build to fail and needed to be removed.

Comment (dherault) [reply www/src/components/account/AccountAttributes.tsx:84]: I made sure the build was passing. Going with honorable v1 made the typing system stricter.

**Pull request merged**